crates/hcl-edit/src/parser/expr.rs

```diff
@@ -199,7 +199,10 @@ fn traversal<'i, 's>(
     move |input: &mut Input<'i>| {
         repeat(
             1..,
-            prefix_decorated(ws_or_sp(state), traversal_operator.map(Decorated::new)),
+            prefix_decorated(
+                ws_or_sp(state),
+                preceded(not(".."), traversal_operator.map(Decorated::new)),
```

Comment:
That would be these:
https://github.com/martinohmann/hcl-rs/blob/03aff259c7d3fbb75ea4a431e64cd75984d6ae7b/crates/hcl-edit/tests/regressions.rs#L280-L294

Since we're already in a "traversal," we have to ensure that there's a way to correctly interpret a `...` correctly rather than assuming it's a `.` traversal operator.

Reply:
Ah right, that makes a lot of sense, thanks!